ipatests/test_integration/test_backup_and_restore.py

```diff
@@ -153,6 +156,18 @@ def backup(host):
         raise AssertionError('Backup directory not found in output')
 
 
+def user_add(host, username):
+    arg = ['ipa', 'user-add', '--first', 'None', '--last', 'None', username]
+    cmd = host.run_command(arg)
```

Comment:
As you're not using `cmd` anywhere in this function you can return it right away.